.github/workflows/deploy-service.yml

```
@@ -141,3 +149,9 @@ jobs:
                 echo "Version $VERSION for $SERVICE not found in the artifacts registry. Skipping!"
                 exit 1
             fi
+
+      - name: Deploy service
+        if: if: ${{ inputs.service != 'web' && inputs.deploy-type == 'binary' }}
+        run: |
+            [[ ${{ inputs.environment }} == "tools" ]] && component/toolbox/awsi.sh upgrade -p tools-prod -r us-east-1 -a y -s ${{ inputs.service }}
+            [[ ${{ inputs.environment }} == "production" ]] && component/toolbox/awsi.sh upgrade -p si-apps-prod -r us-east-1 -a y -s ${{ inputs.service }}
```

If you make the profile an environment-specific env var you can make this one check
```suggestion
            component/toolbox/awsi.sh upgrade -p ${{ env.AWS_PROFILE }} -r us-east-1 -a y -s ${{ inputs.service }}
```

---

Good plan, cheers, will sort